changes/4410-tlambert03.md

```diff
@@ -0,0 +1 @@
+Fix duplicate calls to `__set_name__` for non-private attributes in `BaseModel.__new__`.  Add more tests.
```

Comment:
This file can be removed, if anything, update the change from #4407.

You'll need to add "skip change file check" to the PR body.

Reply:
ok, all set this time for real :joy:  
sorry for the 2-step PR!